app/jobs/assignment_deadline_submission_job.rb

```diff
@@ -3,13 +3,14 @@ class AssignmentDeadlineSubmissionJob < ApplicationJob
 
   def perform(assignment_id)
     assignment = Assignment.find_by(id:assignment_id)
+    @proj_s = 0
+    @proj_ns = 0
```

Comment:
Make variables for descriptive, like projects_sent. proj_sent is ambiguous and difficult to understand what is happening in the code

Reply:
Same for other newly introduced vars

Reply:
Shubhankar wanted me to resolve Hound issues. Many of them were about how long lines were. I had to drastically shorten variable names for that.

Reply:
Should I ignore Hound issues anyway and change the name?

Reply:
Hound is to make code make more readable, do not make code worse to silence hound. You can have longer names if that makes sense in that context, ignore hound errors in such contexts